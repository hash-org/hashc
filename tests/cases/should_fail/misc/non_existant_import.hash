k := import("does_not_exist");
