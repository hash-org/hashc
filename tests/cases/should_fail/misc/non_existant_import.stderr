error: couldn't import `does_not_exist`, module couldn't be found
 --> $DIR/non_existant_import.hash:1:6
1 |   k := import("does_not_exist");
  |        ^^^^^^^^^^^^^^^^^^^^^^^^ here
